esb-connector-creatio-crm/src/main/resources/creatiocrm-case/caseGet.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  ~
+  -->
+<template name="caseGet" xmlns="http://ws.apache.org/ns/synapse">
+<parameter name="id" description="The id of the object will be Get"/>
+<sequence>
+    <property name="callEndpointUri" expression="fn:concat('/0/odata/Case/',$func:id)"/>
+    <property name="httpMethod" value="get"/>
+    <creatiocrm.callWithRetry/>
+</sequence>
```

Comment:
Formatting issue?

Reply:
Please check the other xml files as well, there we few formatting issues

Reply:
Formatted